docs/sphinx/examples/python/qaoa_maxcut.py

```diff
@@ -58,17 +58,21 @@ def kernel_qaoa() -> cudaq.Kernel:
                                                  parameter_count)
 print("Initial parameters = ", optimizer.initial_parameters)
 
-# Pass the kernel, spin operator, and optimizer to `cudaq.vqe`.
-optimal_expectation, optimal_parameters = cudaq.vqe(
-    kernel=kernel_qaoa(),
-    spin_operator=hamiltonian,
-    optimizer=optimizer,
-    parameter_count=parameter_count)
+
+# Define the objective, return `<state(params) | H | state(params)>`
+def objective(parameters):
+    return cudaq.observe(kernel_qaoa, hamiltonian, qubit_count, layer_count,
+                         parameters).expectation()
+
+
+# Optimize!
+optimal_expectation, optimal_parameters = optimizer.optimize(
```

Comment:
What was the reason here for no longer using cudaq.vqe?

Reply:
To be honest, I don't remember updating this file myself but I must have since I don't see it tracked for anyone else on the excel file... So the short answer is I don't know, but I will try to update it now 😃

Reply:
Ok so this isn't as straightforward of an update as I thought it'd be because we now have arguments to our kernel, which need to be mapped by vqe. I can't seem to get that working, so will merge with this and follow-up with a change later